README.md

```diff
@@ -123,16 +123,17 @@ You can turn off SIP on El Capitan this way:
   * Click the  (Apple) menu and select Restart....
 
 The following commands are used to bootstrap CodeChecker on
-OS X El Capitan 10.11.5 and macOS Sierra 10.12 Beta.
-
-Check out and build LLVM/Clang with extra tools. Follow the [Get Started with
-LLVM/Clang](http://clang.llvm.org/get_started.html) documentation.
+OS X El Capitan 10.11, macOS Sierra 10.12 and macOS High Sierra 10.13.
 
 ~~~{.sh}
 # Download and install dependencies.
 brew update
 brew install doxygen gcc git
-brew install homebrew/versions/thrift090
+brew llvm
+
+# Install Thrift 0.9.3 (Note: the general Thrift version is 0.10.0 on macOS High Sierra 10.13)
+brew unlink thrift
+brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/9d524e4850651cfedd64bc0740f1379b533f607d/Formula/thrift.rb
```

Comment:
Couldn't we install thrift [this way](https://github.com/tmsblgh/codechecker/blob/5f035fa55fecefa0a2135fd21b4d9d0f51198c22/.travis.yml#L35) ? In Travis it works.

Reply:
You are right we can, just 'brew install homebrew/versions/thrift090' is not working.